sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1575 @@
+        externalId: identity.displayName
+        type: String
+      - name: applicationValue
+        externalId: application.value
+        type: String
+      - name: applicationRef
+        externalId: application.["$ref"]
+        type: String
+      - name: applicationDisplayName
+        externalId: application.displayName
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  Alert:
+    displayName: SailpointAlert
+    externalId: Alerts
+    description: Entity representing a Sailpoint Identity IQ Alert.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: name
+        externalId: name
+        type: String
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: type
+        externalId: type
+        type: String
+      - name: applicationValue
+        externalId: application.value
+        type: String
+      - name: applicationRef
+        externalId: application.["$ref"]
+        type: String
+      - name: applicationDisplayName
+        externalId: application.displayName
+        type: String
+      - name: alertDate
+        externalId: alertDate
+        type: DateTime
+      - name: lastProcessed
+        externalId: lastProcessed
+        type: DateTime
+      - name: nativeId
+        externalId: nativeId
+        type: String
+      - name: targetId
+        externalId: targetId
+        type: String
+      - name: targetType
+        externalId: targetType
+        type: String
+      - name: targetDisplayName
+        externalId: targetDisplayName
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  # TODO: Identify unique and index attributes
+  AlertAttributes:
+    parent: Alert
+    displayName: SailpointAlertAttributes
+    externalId: attributes
+    description: Entity representing the attibutes associated with the Sailpoint Identity IQ Alert.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
```

Comment:
Same comment as on https://github.com/SGNL-ai/catalog-sgnl-templates/pull/30, I think these should be removed for child entities